config/initializers/dor_services.rb

```diff
@@ -0,0 +1,10 @@
+require 'dor-services'
+
+Dor::Config.dor_services.url ||= Dor::Config.dor.service_root
+
+# TODO: use dor-workflow-service gem (see #194)
+# require 'dor-workflow-service'
```

Comment:
Seems like leaving the `Dor::WorkflowService.configure` lines in gets us closer to #194 than taking them out.  Am I missing something?

Reply:
yes - we SHOULD use dor-workflow-services gem  if we can, rather than dor-services gem -- it clarifies exactly what we are using.  `run_pre_assembly`, the way we are running it, seems to only need a single external lookup:   the workflow service.   I'm not sure if it needs other things from the dor-services gem.

It is currently an open question what external services the discovery report needs -- that may make a case for using the dor-services gem.